.stylelintrc

```diff
@@ -0,0 +1,51 @@
+{
+  "extends": [
+    "stylelint-config-standard-scss",
+    "stylelint-config-idiomatic-order"
+  ],
+  "overrides": [
+    {
+      "files": ["src/assets/scss/*.{css,scss}"],
+      "customSyntax": "postcss-scss",
+      "rules": {
+        "media-feature-range-notation": "prefix",
+        "color-function-notation": "legacy",
+        "shorthand-property-no-redundant-values": null,
+        "declaration-block-no-redundant-longhand-properties": [true, {
+          "ignoreShorthands": [
+            "inset",
+            "inset-block",
+            "inset-inline",
+            "mask",
+            "overscroll-behavior",
+            "padding-block",
+            "padding-inline",
+            "place-content",
+            "place-items",
+            "place-self",
+            "scroll-margin",
+            "border-block",
+            "border-block-end",
+            "border-block-start",
+            "border-inline",
+            "border-inline-end",
+            "border-inline-start"
+          ]
+        }],
+        "no-invalid-position-at-import-rule": null,
+        "no-descending-specificity": null,
+        "selector-class-pattern": null,
+        "scss/dollar-variable-pattern": null,
+        "selector-id-pattern": null,
+        "scss/no-global-function-names": null,
+        "selector-type-no-unknown": null,
+        "property-no-vendor-prefix": [true,
+          {
+            "ignoreProperties": ["backdrop-filter", "user-select", "initial-letter", "text-decoration", "text-stroke", "text-fill-color", "line-clamp", "box-decoration-break", "stretch"]
+          }
+        ],
```

Comment:
Is this config removing all vendor prefixes except for those on the properties listed on line 44?

Reply:
Yes it is

Reply:
These properties still need prefixes to be supported on all major browsers.